### ISSUE TITLE
[Refactoring]: fix cases marked by linter `errcheck` for package `github.com/matrixorigin/matrixone/pkg/container`

### ISSUE THREAD

**Issue by cnutshell** [matrixorigin/matrixone#2525]
### Is there an existing issue for the refactoring request?

- [X] I have checked the existing issues.

### Is your refactoring request related to a problem?

```markdown
We should check all the cases marked by linter errcheck.
Errcheck is a program for checking for unchecked errors in go programs. 
These unchecked errors can be critical bugs in some cases.
```

### Describe the solution you'd like

Check all the cases marked by linter errcheck and evaluate them.
If an unchecked error can be critical bugs in some cases, then fix it.

### Additional information

How to find all the cases via `errcheck` for package `github.com/matrixorigin/matrixone/pkg/container`:
```bash
git clone https://github.com/matrixorigin/matrixone.git
cd matrixone
## This will show all the cases for both normal code and test code
golangci-lint run --disable-all -E errcheck \
    --max-same-issues 10000 --max-issues-per-linter 10000 pkg/container/...
```

If you want to find cases for non-test code only, using option `--skip-files ".+_test.go"`:
```bash
golangci-lint run --disable-all -E errcheck \
    --max-same-issues 10000 --max-issues-per-linter 10000 --skip-files ".+_test.go" pkg/container/...
```

The output of this command would be:
```bash
pkg/container/ring/approxcd/approx.go:162:15: Error return value of `(*github.com/axiomhq/hyperloglog.Sketch).Merge` is not checked (errcheck)
	r.Sk[x].Merge(ar.Sk[y])
	             ^
```

**Comment by cnutshell** [comment]
Related with issue #2133

**Comment by nnsgmsone** [comment]
fixed by 2541

**Comment by cnutshell** [comment]
Resolved in `00050728be`, close this issue.
